access/legacy/convert/convert.go

```diff
@@ -77,7 +77,7 @@ func MessageToTransaction(m *entitiesproto.Transaction, chain flow.Chain) (flow.
 func TransactionToMessage(tb flow.TransactionBody) *entitiesproto.Transaction {
 	proposalKeyMessage := &entitiesproto.Transaction_ProposalKey{
 		Address:        tb.ProposalKey.Address.Bytes(),
-		KeyId:          uint32(tb.ProposalKey.KeyID),
+		KeyId:          uint32(tb.ProposalKey.KeyIndex),
```

Comment:
are we okay with changing the keyID name to KeyIndex, I remember @psiemens had some reasons to use ID instead of Index.

Reply:
@ramtinms I originally made a case for Key ID but we've since moved back to Key Index. We're good to change it anywhere it occurs